### ISSUE TITLE
Explain what is meant by "define our models and assets"

### ISSUE THREAD

**Issue by ttmc** [bigchaindb/js-driver-orm#3]
Explain more about the lines:
```javascript
// define our models and assets
bdbOrm.define("myModel", "https://schema.org/v1/myModel")
```
in `README.md`

What does that mean? How is that information used?

**Comment by diminator** [comment]
good point

`asset` data is fixed and cannot be update, unlike `metadata`. So the "sticky" definition of an asset (like the schema, namespace, etc) can go into the `asset` field. Updateable variables go into `metadata`.

A model is a class of assets that have the same `asset` definition. They belong to the same `class`. The actual `CREATE` transactions create instances of those models/classes.

proposal can be to be more verbose:
```
bdbOrm.define({
    model: "myModel",
    asset: { uri: "https://schema.org/v1/myModel" }
})
```
- Note: `asset` could take more complex objects. 
- Note: for the ORM we don't need to use the same terminology as the underlying driver
ie `asset` => `schema` and `metadata` => `document` cc @r-marques

**Comment by jernejpregelj** [comment]
To me the main thing is do we present asset field to users as an optional field where they can describe or define properties of model (or something else if they like) or should we enforce it and define some structure? 

- Note: users can get pretty creative if we let this more open. (Maybe someone will put in something like html/css template of how model should be displayed... or other crazy stuff)

**Comment by jernejpregelj** [comment]
Addressed in #5
